Kernel/Devices/PATAChannel.cpp

```diff
@@ -1,14 +1,15 @@
-#include <Kernel/Devices/IDEDiskDevice.h>
+#include <AK/ByteBuffer.h>
+#include <Kernel/Devices/PATAChannel.h>
 #include <Kernel/FileSystem/ProcFS.h>
 #include <Kernel/IO.h>
-#include <Kernel/Arch/i386/PIC.h>
 #include <Kernel/Process.h>
-#include <Kernel/StdLib.h>
 #include <Kernel/VM/MemoryManager.h>
+#include "PATADiskDevice.h"
 
-//#define DISK_DEBUG
+#define PATA_MASTER_IRQ 14
```

Comment:
`PATA_PRIMARY_IRQ` and `PATA_SECONDARY_IRQ`, no?

Reply:
Whoops